File: C/Get the Percentage.c
#include <stdio.h>
int main()
{
	double num1,num2,result;
	printf("Enter number1/_\n");
	scanf("%lf",&num1);
	printf("Enter number2/_\n");
	scanf("%lf",&num2);
	result = (num2/num1)*100;
	printf("Result = %.lf%%",result);
	return 0;
}

File: Java/Get the percentage.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		double num1, num2, result;
		Scanner input = new Scanner(System.in);
		System.out.println("Enter number1/_");
		num1 = input.nextDouble();
		System.out.println("Enter number2/_");
		num2 = input.nextDouble();
		result = (num2/num1)*100;
		System.out.println("Result = "+result+"%");
	}
}